book/src/bindings/elixir.md

```diff
@@ -0,0 +1,40 @@
+# PRQL
```

Comment:
If this is just the readme, let's include with `{{ include` like in https://github.com/PRQL/prql/blob/main/book/src/language-features/standard-library.md?plain=1#L20, and then it'll stay updated?

Reply:
aa wait

Reply:
@max-sixty can you do a build on docs? i cant do it in my PC due to the python error i mentioned before

Reply:
Yeah, I just labeled it so it'll deploy to the main docs, easy to undo if it looks bad

Reply:
https://github.com/PyO3/pyo3/issues/2731 this issue keeps hitting me :(

Reply:
![image](https://user-images.githubusercontent.com/13379595/213377862-178e53e3-b458-455c-adea-9c31d62ec0dc.png)

Reply:
You have a 32bit python interpreter?! :)

Reply:
Yeah, need to reinstall but im bit lazy :D